Source/Scene/PostProcessLibrary.js

```diff
@@ -0,0 +1,231 @@
+        /**
+         * Render with a night vision effect.
+         *
+         * @memberof PostProcessLibrary
+         *
+         * @type {PostProcess}
+         * @readonly
+         */
+        nightVision : {
+            get : function() {
+                return createNightVisionStage();
+            }
+        },
+        /**
+         * Overlay a texture above the render.
+         *
+         * @memberof PostProcessLibrary
+         *
+         * @type {PostProcess}
+         * @readonly
+         */
+        textureOverlay : {
+            get : function() {
+                return createTextureOverlayStage();
+            }
+        },
+        /**
+         * @private
+         */
+        depthView : {
+            get : function() {
+                return createDepthViewStage();
+            }
+        },
+        /**
+         * @private
+         */
+        blur : {
+            get : function() {
+                return createBlur();
+            }
+        },
+        /**
+         * private
+         */
+        depthOfField : {
+            get : function() {
+                return createDepthOfFieldStage();
+            }
+        },
+        /**
+         * lensFlare.
+         *
+         * @memberof PostProcessLibrary
+         *
+         * @type {PostProcess}
+         * @readonly
+         */
+        lensFlare : {
+            get : function() {
+                return createLensFlareStage();
+            }
+        },
+        /**
+         * private
+         */
+        silhouette : {
+            get : function() {
+                return createSilhouetteStage();
+            }
+        }
+    });
+
+    function createBlackAndWhiteStage() {
+        return new PostProcess({
+            name : 'czm_black_and_white',
+            fragmentShader : BlackAndWhite,
+            uniformValues : {
+                gradations : 5.0
+            }
+        });
+    }
+
+    function createBrightnessStage() {
+        return new PostProcess({
+            name : 'czm_brightness',
+            fragmentShader : Brightness,
+            uniformValues : {
+                brightness : 0.5
+            }
+        });
+    }
+
+    function createEightBitStage() {
+        return new PostProcess({
+            name : 'czm_eight_bit',
+            fragmentShader : EightBit
+        });
+    }
+
+    function createNightVisionStage() {
+        return new PostProcess({
+            name : 'czm_night_vision',
+            fragmentShader : NightVision
+        });
+    }
+
+    function createTextureOverlayStage() {
+        // not supplying a name means more than one effect can be added
+        return new PostProcess({
+            fragmentShader : TextureOverlay,
+            uniformValues : {
+                alpha : 0.5,
+                // data uri for a 1x1 white canvas
+                texture : 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQYV2P4////fwAJ+wP9BUNFygAAAABJRU5ErkJggg=='
```

Comment:
Is it better to use [Context.defaultTexture](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Renderer/Context.js#L714)?

Reply:
I didn't want this to be the only function that requires a context parameter.

Another option is to add it to the constructor. Wouldn't this go against the coding conventions? Classes in `Renderer` can require a context when created, but classes in scene only get a reference to context as a function parameter.

Reply:
I believe some functions in Scene take a `scene` parameter.  It's also OK to just leave this; it is minor and the default texture is not the best long-term solution - most users want it to be transparent, not solid white.